### PR TITLE
SHPopup

### PR DESCRIPTION
**Pull request by iamshezad**
## Project URL
https://github.com/iamshezad/SHPopup

## Category
popup

## Description
A lightweight library for popup view
 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [ ] Supports Swift 4 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by iamshezad** [comment]
@arasu01 Can u just check this???

File: README.md [hunk 1]
```diff
@@ -2135,7 +2135,10 @@ Most of these are paid services, some have free tiers.
 * [AZDialogViewController](https://github.com/Minitour/AZDialogViewController) - A highly customizable alert dialog controller that mimics Snapchat's alert dialog. :large_orange_diamond:
 * [MIBlurPopup](https://github.com/MarioIannotta/MIBlurPopup) - MIBlurPopup let you create amazing popups with a blurred background.
 * [LNPopupController](https://github.com/LeoNatan/LNPopupController) - a framework for presenting view controllers as popups of other view controllers, much like the Apple Music and Podcasts apps.
-* [PopupWindow](https://github.com/shin8484/PopupWindow) - PopupWindow is a simple Popup using another UIWindow in Swift. :large_orange_diamond:
+* [PopupWindow](https://github.com/shin8484/PopupWindow) - PopupWindow is a simple Popup using another UIWindow in Swift.
+:large_orange_diamond:
```
Comment (lfarah) [inline README.md]: This should be in the same line as the description

File: README.md [hunk 2]
```diff
@@ -2135,7 +2135,10 @@ Most of these are paid services, some have free tiers.
...
+* [SHPopup](https://github.com/iamshezad/SHPopup) - A lightweight library for popup view.
+:large_orange_diamond:
```
Comment (lfarah) [inline README.md]: This should be in the same line as the description

**Comment by lfarah** [comment]
@iamshezad thanks for contributing! Let's just do some simple fixes so we can approve this ASAP

**Comment by lfarah** [comment]
@danger-awesome-ios flagged you because you made too many changes. We appreciate that you updated the links, but in the next time, that should be done in 2 different PRs to keep things organized, so the rule is: if you're adding a new library, only add 1 line to the README.

**Comment by lfarah** [comment]
Also, do you support Cocoapods? It would be good to mention it in the README [just like this](https://github.com/awesome-labs/iOS-readme-template#installation)

**Comment by iamshezad** [comment]
@lfarah 
Thank you for considering!
Yes it supports Cocoapods but the older version so i didn't mentioned it and i wanted it to be as a lightweight library.

**Comment by iamshezad** [comment]
@lfarah Issue fixed

**Comment by iamshezad** [comment]
@lfarah @arasu01 what should i do to solve these issues???

**Comment by arasu01** [comment]
Everything seems ok @iamshezad except this rule
 **Only one project/change is on this pull request.

**Comment by iamshezad** [comment]
@arasu01 
Thank you for your response.
I will close this pull request and create a new one as per your suggestion.

**Pull request closed without merging**